Library/Homebrew/cmd/bottle.rb

```diff
@@ -33,25 +33,19 @@
 EOS
 
 module Homebrew
-  def keg_contains string, keg
-    if not ARGV.homebrew_developer?
-      return quiet_system 'fgrep', '--recursive', '--quiet', '--max-count=1', string, keg.to_s
-    end
-
+  def keg_contains string, keg, options={}
     result = false
     index = 0
 
     keg.each_unique_file_matching(string) do |file|
-      if ARGV.verbose?
-        opoo "String '#{string}' still exists in these files:" if index.zero?
-        puts "#{Tty.red}#{file}#{Tty.reset}"
-      end
```

Comment:
So, the problem I found with this was that it was effectively duplicated by the output from below. I'm open to preserving it but I couldn't see a way of doing so.